examples/azure-vm-simple-linux-managed-disk/README.md

```diff
@@ -0,0 +1,24 @@
+# Very simple deployment of a Linux VM
+
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-vm-simple-linux%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
```

Comment:
@scottzilla sorry, to clarify that: is there a similar tool for displaying Terraform configuration files, rather than an ARM Template? If not, I think it'd make sense to remove this, rather than confuse users?